src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -889,6 +893,13 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
     };
   });
 
+  const setGasPanelHeight = useCallback(
+    (height: number) => {
+      gasPanelHeight.value = height;
+    },
+    [gasPanelHeight]
+  );
+
```

Comment:
we probably don't need this since it's just setting a SharedValue directly. I'd much rather keep this file clean and just inline the change where we need to.